source/databricks/calculation_engine/tests/calculation_input/test_table_reader.py

```diff
@@ -11,13 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import pathlib
 from datetime import datetime
 from decimal import Decimal
 from unittest import mock
 import pytest
 from pyspark.sql import SparkSession, DataFrame
-from pyspark.sql.functions import lit
+from pyspark.sql.types import StructType
+from pyspark.sql.functions import col, when, lit
```

Comment:
FYI: Jonas and I are moving towards
```suggestion
import pyspark.sql.functions as f
```